### PR TITLE
0.9.4-beta release

### PR DESCRIPTION
**Pull request by gauchm**
Changelog:
- Added new config arguments for custom normalization (`custom_normalization`), duplication of features (`duplicate_features`) and lagged features (`lagged_features`)
- Added tutorial for the MTS-LSTM architecture
- Added tutorial for adding a new model
- Added tutorial for adding a new dataset
- Added new mode `finetune`. As the name suggests, this mode can be used to finetune a pre-trained model on a new dataset. It is possible to only finetune parts of the model and freeze all other parts. Supports all currently available models. New config argument was added (`finetune_modules`) to specify which model parts are updated during finetuning
- Added page for config arguments to documentation
- Updated documentation landing page

### CODE REVIEW

**Review by kratzert — APPROVED** [review]
LGTM
